pyproject.toml

```diff
@@ -28,7 +28,7 @@ dependencies = [
   "html5lib",
   "nbconvert",
   "nbformat",
-  "pytest >=7.0,<9",
+  "pytest >=8.1,<9",
```

Comment:
FYI: The changes in this PR are compatible with Pytest 7.0+ ; you don't have to bump the minimum version.